cmd/helm/lint.go

```diff
@@ -91,12 +91,11 @@ func newLintCmd(out io.Writer) *cobra.Command {
 
 			fmt.Fprintf(out, message.String())
 
-			var summary strings.Builder
-			fmt.Fprintf(&summary, "%d chart(s) linted, %d chart(s) failed", len(paths), failed)
+			summary := fmt.Sprintf(&summary, "%d chart(s) linted, %d chart(s) failed", len(paths), failed)
```

Comment:
correct usage would be `summary := fmt.Sprintf("%d chart(s) linted[....]`.

Reply:
Doh, I knew I should have double checked before I pushed. Have fixed this now.